private/data/i18n/en.json

```diff
@@ -292,7 +292,18 @@
         "orderSetToState": "{{orderNumber}} {{orderText}} set to {{status}}",
         "skippedBulkOrdersAlert": "You've requested that {{selectedOrders}} {{orderText}} be set to the '{{status}}' status, but {{numberOfSkippedOrders}} of these orders {{skippedOrdersText}} not in the '{{skippedState}}' state and would skip all steps leading up to the '{{status}}' state. Are you sure you want to do this?",
         "bulkOrdersRegressionAlert": "You're about to regress {{ordersToRegress}} {{orderText}} back to the '{{status}}' state. Are you sure you want to continue with this operation?",
-        "paymentCaptureSuccess": "Payments successfully captured"
+        "paymentCaptureSuccess": "Payments successfully captured",
+        "table": {
```

Comment:
We need to be putting these i18n definitions in the `/server/i18n` directory within the plugin and not in the project wide `/private/data/i18n` file. I'll fix this one, but let's make sure that from here on out, the i18n defs are in the plugins directory whenever they are used by a plugin